docs/07-create-addons-repo.md

```diff
@@ -0,0 +1,296 @@
+---
+title: Create addons repository
+type: Details
+---
+
+The repository in which you create your own addons must contain at least one `index.yaml` file and have a specific structure, depending on the type of server that exposes your addons.
+
+## The index yaml file
+
+Your remote addons repository can contain many addons defined in index files. Depending on your needs and preferences, you can create one or more index files to categorize your addons. In the index file, provide an entry for every single addon from your addons repository. The index file must have the following structure:
+```
+apiVersion: v1
+entries:
+  {addon_name}:
+    - name: {addon_name}
+      description: {addon_description}
+      version: {addon_version}
+```
+
+See the example:
+```yaml
+apiVersion: v1
+entries:
+  redis:
+    - name: redis
+      description: Redis service
+      version: 0.0.1
+```
+
+>**NOTE:** You must place your addons in the same directory where the `index.yaml` file is stored.
+
+## Supported protocols
+
+Expose your addons repository so that you can provide URLs in the [AddonsConfiguration](https://github.com/kyma-project/helm-broker/blob/master/docs/13-cr-addonsconfiguration.md) (AC) and [ClusterAddonsConfiguration](https://github.com/kyma-project/helm-broker/blob/master/docs/14-cr-clusteraddonsconfiguration.md) (CAC) custom resources. The Helm Broker supports exposing addons using the following protocols:
```

Comment:
```suggestion
Expose your addons repository so that you can provide URLs in the [AddonsConfiguration](./13-cr-addonsconfiguration.md) (AC) and [ClusterAddonsConfiguration](./14-cr-clusteraddonsconfiguration.md) (CAC) custom resources. Helm Broker supports exposing addons using the following protocols:
```